File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.size() != t.size()){
            return false;
        }

        map<char,int>ss;
        map<char,int>tt;

        for(auto i : s){
            ss[i]++;
        }
        for(auto i : t){
            tt[i]++;
        }

        return ss == tt;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t):
            return False

        ss = defaultdict(int)
        tt = defaultdict(int)

        for i in s:
            ss[i]+=1

        for i in t:
            tt[i]+=1

        return ss == tt    


